qutebrowser/utils/utils.py

```diff
@@ -864,6 +864,63 @@ def unused(_arg):
     pass
 
 
+def parse_numsettxt_into_list(txtset, nummax):
```

Comment:
Does qutebrowser actually hang because of this when doing something stupid like `:download-delete 1,3,5-100000`? if not: Don't optimize it if it ain't broken :wink: